### PR TITLE
Add a filter for serialization attributes

### PR DESCRIPTION
**Pull request by meyerbaptiste**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #1083 
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php [hunk 1]
```diff
@@ -37,11 +34,6 @@ class ExistsFilter extends AbstractFilter
 {
     const QUERY_PARAMETER_KEY = 'exists';
 
-    public function __construct(ManagerRegistry $managerRegistry, RequestStack $requestStack, LoggerInterface $logger = null, array $properties = null)
```
Comment (soyuka) [inline src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php]: I've seen this in other filters I think. Do this need to be part of this commit?

Comment (meyerbaptiste) [inline src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php]: No this is the only filter where there is this useless constructor! I will open another PR.

Comment (soyuka) [inline src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php]: Okay, we have other useless constructors (QueryChecker for example)

Comment (meyerbaptiste) [inline src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php]: Not totally useless, the `QueryChecker`'s constructor is private! 😛

**Comment by teohhanhui** [comment]
Perhaps "properties" instead of "fields" to be consistent in our terminology?

**Comment by teohhanhui** [comment]
But I guess people are more used to "fields"... Maybe we should just keep it this way. :smile:

**Comment by dunglas** [comment]
I'm for `property`. It's why what we use in API Platform (`@ApiProperty`) and it's the Hydra terminology.

**Comment by meyerbaptiste** [comment]
Done.

**Comment by teohhanhui** [comment]
> It's why what we use in API Platform (`@ApiProperty`) and it's the Hydra terminology.

Yes, but what would people be looking for, when they want that feature? "Fields". It might be worth using the term most people are likely most familiar with.

**Comment by dunglas** [comment]
> Yes, but what would people be looking for, when they want that feature? "Fields". It might be worth using the term most people are likely most familiar with.

It's a doc issue. We need to make it easy to find on the website using both terms, but in the code, consistency is the key. Keep it as is.

**Comment by dunglas** [comment]
Good job!!

**Pull request merged**